dallinger/experiment_server/experiment_server.py

```diff
@@ -220,17 +226,39 @@ def launch():
     """Launch the experiment."""
     exp = Experiment(db.init_db(drop_all=False))
     exp.log("Launching experiment...", "-----")
-    url_info = exp.recruiter().open_recruitment(n=exp.initial_recruitment_size)
-    session.commit()
+    try:
+        url_info = exp.recruiter().open_recruitment(n=exp.initial_recruitment_size)
+        session.commit()
+    except:
```

Comment:
The bare excepts here make me feel a little nervous. It'd almost certainly be fine for real-life cases, but in the tests below you clean up the process by sending ctrl+c, so there's an unlikely race condition here where one of these try/excepts capture the ctrl+c and the process continues.

Can we do `except Exception` instead?